feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/service/storage/impl/MetisStorageServiceImpl.kt

```diff
@@ -146,12 +146,13 @@ internal class MetisStorageServiceImpl(
             serverId: String,
             clientSidePostId: String,
             serverSidePostId: Long?,
+            conversationId: Long?,
```

Comment:
I think this could would be cleaner if the parameter would be ```conversationId: Long = this.conversationId```. Then you do not need the ?: in the body.